src/library/Uno.Cupertino/Styles/Controls/ProgressRing.xaml

```diff
@@ -5,35 +5,30 @@
 					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 					xmlns:not_win="http://uno.ui/not_win"
 					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:IsNotWinUI3="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsTypeNotPresent(Microsoft.UI.Xaml.FrameworkElement)"
 					xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+					xmlns:lottie_win="using:Microsoft.Toolkit.Uwp.UI.Lottie?IsTypeNotPresent(Uno.UI.FeatureConfiguration)"
+					xmlns:lottie_not_win="using:Microsoft.Toolkit.Uwp.UI.Lottie?IsTypePresent(Uno.UI.FeatureConfiguration)"
```

Comment:
Neat

Reply:
Neat, but only with with UWP, not WinUI :(

![image](https://user-images.githubusercontent.com/5839577/146201615-513d871a-be93-4f26-a202-0abdbc384271.png)